File: AutoShutdown/AutoShutdown/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Diagnostics;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace AutoShutdown
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {

        }


        private void timer1_Tick(object sender, EventArgs e)
        {
            label2.Text = DateTime.Now.ToString();
            int st = 0;
            if(checkBox1.Checked == true)
                st += (ConvertToInt(textBox1.Text) - ConvertToInt(DateTime.Now.Hour.ToString()) + 24) * 3600;
            else
                st += (ConvertToInt(textBox1.Text) - ConvertToInt(DateTime.Now.Hour.ToString())) * 3600;

            st += (ConvertToInt(textBox2.Text) - ConvertToInt(DateTime.Now.Minute.ToString())) * 60;
            if(st < 0)
                label6.Text = "时间小于当前时间";
            else
                label6.Text = st.ToString();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            int st = 0;
            if (checkBox1.Checked == true)
                st += (ConvertToInt(textBox1.Text) - ConvertToInt(DateTime.Now.Hour.ToString()) + 24) * 3600;
            else
                st += (ConvertToInt(textBox1.Text) - ConvertToInt(DateTime.Now.Hour.ToString())) * 3600;
            st += (ConvertToInt(textBox2.Text) - ConvertToInt(DateTime.Now.Minute.ToString())) * 60;

            Process myProcess = new Process();
            Process.Start("cmd.exe","/k shutdown -s -t " + st.ToString());

        }
        private int ConvertToInt(string InStr)
        {
            try
            {
                return Convert.ToInt32(InStr);
            }
            catch
            {
                return 0; 
            }
        }
    }
}

File: 假装有用系列/刷q钻/刷q钻/Form1.vb
Public Class Form1

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        If (Val(TextBox1.Text) <= 10000) Then
            MsgBox("请输入正确的QQ帐号！")
        Else
            MsgBox("点击确认开始刷钻，爆破可能会进行很长时间，爆破速度取决于网速以及电脑性能")
            Threading.Thread.Sleep(8000)
            Form2.Label2.Text = TextBox1.Text
            Form2.Show()
        End If
    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Form3.Show()
        Threading.Thread.Sleep(4000)
        Form3.Label1.Text = "已是最新版本"
    End Sub
End Class
